pkgs/servers/osmocom/osmo-bts/default.nix

```
@@ -43,5 +43,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.agpl3Plus;
     maintainers = with lib.maintainers; [ janik ];
     platforms = lib.platforms.linux;
+    mainProgram = "osmo-bts";
```

```suggestion
    mainProgram = "osmo-bts-virtual";
```
there is no osmo-bts, just osmo-bts-virtual and osmo-bts-omldummy. You probably want osmo-bts-virtual so I'd write that here.

---

You are right. I overlooked that. Since, there are multiple options and it depends on which back-end you want use, does mainProgram even make sense here?

---

Yeah I guess not having a mainProgram is sensible in this case.